src/main/app/claims/ccjModelConverter.ts

```diff
@@ -37,22 +36,22 @@ function convertPayBySetDate (draftCcj: DraftCCJ): Moment {
 
 export class CCJModelConverter {
 
-  static convert (draftCcj: DraftCCJ): CountyCourtJudgment {
+  static convert (draft: DraftCCJ): CountyCourtJudgment {
     let statementOfTruth: StatementOfTruth = undefined
-    if (draftCcj.qualifiedDeclaration) {
+    if (draft.qualifiedDeclaration) {
       // API model is called statement of truth
```

Comment:
is this important comment?

Reply:
Hard to answer, @timja-kainos felt like it gives a value.